examples/hyperbolic/hypertree.ml

```diff
@@ -583,47 +583,21 @@ let from_screen canvas x y =
 
 let pi = 4. *. atan 1.
 
-let ellipse_arc c cx cy rx ry start fin clock_wise =
-  c##save;
-  c##translate (Js.float cx) (Js.float cy);
-  c##scale (Js.float rx) (Js.float ry);
-  c##arc
-    (Js.float 0.)
-    (Js.float 0.)
-    (Js.float 1.)
-    (Js.float start)
-    (Js.float fin)
-    clock_wise;
-  c##restore
-
 let arc c (rx, ry, dx, dy) z0 z1 z2 =
   let rd = norm (sub z1 z0) in
   let start = atan2 (z1.y -. z0.y) (z1.x -. z0.x) in
   let fin = atan2 (z2.y -. z0.y) (z2.x -. z0.x) in
   c##beginPath;
   let alpha = mod_float (fin -. start +. (2. *. pi)) (2. *. pi) in
-  (*
-Firebug.console##log_4(start, fin, alpha, (alpha > pi));
-*)
-  if rx = ry
-  then
-    c##arc
-      (Js.float ((z0.x *. rx) +. dx))
-      (Js.float ((z0.y *. rx) +. dy))
-      (Js.float (rd *. rx))
-      (Js.float start)
-      (Js.float fin)
-      (Js.bool (alpha > pi))
-  else
-    ellipse_arc
-      c
-      ((z0.x *. rx) +. dx)
-      ((z0.y *. ry) +. dy)
-      (rd *. rx)
-      (rd *. ry)
-      start
-      fin
-      (Js.bool (alpha > pi));
+  c##ellipse
+    ((z0.x *. rx) +. dx)
```

Comment:
Some float conversions are missing here.

Reply:
See https://github.com/ocaml-wasm/wasm_of_ocaml/pull/57/commits/1054d571aea084ad25233f6b7457f020230eb9b0

Reply:
Well spotted, I added the missing commit.